gov_uk_dashboards/formatting/human_readable.py

```diff
@@ -1,23 +1,46 @@
 """Functions for converting values to human readable format"""
 
 
-def format_as_human_readable(value_to_format: float, *, prefix: str = "") -> str:
-    """Format a number as a human readable string
-    Appends bn, m, k as appropriate
-    Rounds number to 3dp
+from typing import Optional
+
+
+def format_as_human_readable(
```

Comment:
minor point - but do we need the word human in this?

Reply:
Happy to take other options on the name-  it's called that because human-readable is a standard term referring to data that if formatted in a way to make it quick and easy for humans to read. Human is specified because data can also be formatted in ways to make it more machine-readable.

Reply:
i see, i thought it meant something else. learn something new everyday